server/src/main/java/module-info.java

```diff
@@ -368,4 +369,5 @@
     uses org.elasticsearch.reservedstate.ReservedClusterStateHandlerProvider;
 
     provides org.apache.lucene.codecs.PostingsFormat with org.elasticsearch.index.codec.bloomfilter.ES85BloomFilterPostingsFormat;
+    provides org.apache.lucene.codecs.DocValuesFormat with org.elasticsearch.index.codec.tsdb.ES97TSDBDocValuesFormat;
```

Comment:
the name should be ES87 not ES97 I believe since that would be introduced in Elasticsearch 8.7?

Reply:
Of course 🤦